src/A2A/Models/AgentCapabilities.cs

```diff
@@ -7,6 +7,21 @@ namespace A2A;
 /// </summary>
 public sealed class AgentCapabilities
 {
+    /// <summary>
+    /// Creates a new instance of the <see cref="AgentCapabilities"/> class.
+    /// </summary>
+    public AgentCapabilities()
+    {
+    }
+
+    internal AgentCapabilities(AgentCapabilities source)
```

Comment:
I introduced copy ctors so the next time we add a new property, it will be easier to discover that this logic needs to be updated.

Reply:
just curious, is `Clone()` still a thing in 2025? Should we opt for those semantics instead of `new X(oldX)`?